job/server/pom.xml

```diff
@@ -52,6 +52,11 @@
       <artifactId>jersey-container-servlet-core</artifactId>
       <version>${jersey.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.jaxrs</groupId>
+      <artifactId>jackson-jaxrs-json-provider</artifactId>
+      <version>${jackson.version}</version>
+    </dependency>
```

Comment:
this dependency and version should be specified in the root pom's `<dependencyManagement>` section, and the version can be left out here.

Reply:
done